framework/src/main/java/org/checkerframework/framework/util/FlowExpressionParseUtil.java

```diff
@@ -158,7 +145,73 @@ private static Pattern anchored(String regex) {
     }
 
     private static boolean isMemberSelect(String s, FlowExpressionContext context) {
-        return parseMemberSelect(s) != null;
+        return parseMemberSelect(s, context) != null;
+    }
+
+    /**
+     * Matches a field access which contains a method call either in receiver or in remaining. First
```

Comment:
Can you give an example of what the expression is?  I don't understand the method's specification.